internal/pkg/print/print_test.go

```diff
@@ -55,16 +64,26 @@ func TestOutputf(t *testing.T) {
 				Verbosity: tt.verbosity,
 			}
 
+			if tt.outputFormatNone {
+				viper.Set(config.OutputFormatKey, NoneOutputFormat)
+			} else {
+				viper.Set(config.OutputFormatKey, "") // needed to prevent conflict with other tests
```

Comment:
I think if we use the [`viper.New`](https://pkg.go.dev/github.com/dvln/viper#New) method this won't be needed